save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -63,59 +71,205 @@ external interface ProjectViewState : RState {
     /**
      * Whether error modal from backend is visible
      */
-    var isErrorOpen: Boolean
+    var isErrorFromBackend: Boolean
 
     /**
      * Whether error text from backend is visible
      */
-    var errorText: String
+    var errorTextFromBackend: String
+
+    /**
+     * Flag to check for git url
+     */
+    var isNoGitUrl: Boolean
+
+    /**
+     * Flag to check that project have been presented as url
+     */
+    var isProjectAsGitUrl: Boolean
+
+    /**
+     * Path to property file
+     */
+    var pathToProperty: String?
+
+    /**
+     * Url of project's git
+     */
+    var gitUrlProject: String?
+
+    /**
+     * Flag to check that project have been presented as binary file
+     */
+    var isProjectAsBinaryFile: Boolean
+
+    /**
+     * Flag to check that both cards have been opened
+     */
+    var isBothProjectOpen: Boolean
+
+    /**
+     * Flag to check that none cards have been opened
+     */
+    var isNoneProjectOpen: Boolean
+
+    /**
+     * Flag to check that binary file haven't been selected
+     */
+    var isBinaryFileNotSelect: Boolean
```

Comment:
These checks can be done in form's `onSumbit` handler, why do you need them in state?